### PR TITLE
Fix empty instruments and templates list on Windows

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
Fixes
imageand
image

### CODE REVIEW

File: src/framework/global/internal/globalconfiguration.cpp [hunk 1]
```diff
@@ -74,7 +74,7 @@ QString GlobalConfiguration::resolveAppDataPath() const
 #endif
 
 #ifdef Q_OS_WIN
-    QDir dir(QCoreApplication::applicationDirPath() + QString("/../" MUSE_APP_INSTALL_NAME));
+    QDir dir(QCoreApplication::applicationDirPath() + QString("/../"));
```
Comment (igorkorsukov) [inline src/framework/global/internal/globalconfiguration.cpp:77]: It was like that before my changes, I just renamed it.
```
QDir dir(QCoreApplication::applicationDirPath() + QString("/../" MUSESCORE_INSTALL_NAME));
```

And also keep for other platforms
```
QDir dir(QCoreApplication::applicationDirPath() + QString("/../share/" MUSESCORE_INSTALL_NAME));
```
https://github.com/musescore/MuseScore/blob/4.3.0/src/framework/global/internal/globalconfiguration.cpp#L68

Have you done the install step? Without it, the tools were always not displayed

Comment (Jojo-Schmitz) [reply src/framework/global/internal/globalconfiguration.cpp:77]: Yes, of course I did the install step (but yes, such stupid mistakes are possible ;-)). But `MUSESCORE_INSTALL_NAME` is not the same as `MUSE_APP_INSTALL_NAME`, the latter contains "App", but no such directory exists on Windows

Comment (Jojo-Schmitz) [reply src/framework/global/internal/globalconfiguration.cpp:77]: And I'm not saying that my change is correct, just that it fixes the issue, on Windows.

Comment (igorkorsukov) [reply src/framework/global/internal/globalconfiguration.cpp:77]: Hmm, ok, thanks, I'll merge this fix and later I'll see what actually happened

**Pull request merged**